lightningd/pay.c

```diff
@@ -160,22 +160,22 @@ static bool send_payment(struct command *cmd,
 		/* FIXME: We should really do something smarter here! */
 		log_debug(cmd->ld->log, "json_sendpay: found previous");
 		if (payment->status == PAYMENT_PENDING) {
-			log_add(cmd->ld->log, "... still in progress");
-			command_fail(cmd, "still in progress");
+			log_add(cmd->ld->log, "Payment is still in progress");
```

Comment:
BTW log_add literally appends to the previous log entry, so the ... was correct.  But that's totally cosmetic...

Reply:
Maybe we should change this so that instead of using log_add it uses log_(something I can't remember right now) so it adds it as a new line instead of as an append?